src/tools/compiletest/src/main.rs

```diff
@@ -10,9 +10,7 @@
 
 #![crate_name = "compiletest"]
 
-#![feature(box_syntax)]
 #![feature(test)]
```

Comment:
I'd like to get rid of the dependence of compiletest on the `test` crate entirely. I don't really see any reason for this dependency to exist except some minor code sharing. The major blocker is the `run_test` function that is called in the `test` crate.